fe/fe-core/src/main/java/com/starrocks/catalog/OlapTable.java

```diff
@@ -909,6 +909,14 @@ public Collection<Partition> getAllPartitions() {
         return partitions;
     }
 
+    public List<Long> getShardGroupIds() {
```

Comment:
shard group is for lake table, so better to implement this in LakeTable.

Reply:
it will generate runtime cast error if we do this, so it must be put here